judge/jinja2/markdown/__init__.py

```diff
@@ -117,8 +118,11 @@ def get_cleaner(name, params):
     if name in cleaner_cache:
         return cleaner_cache[name]
 
-    if params.get('styles') is True:
-        params['styles'] = all_styles
+    styles = params.pop('styles', None)
+    if styles is True:
+        styles = all_styles
+    if styles is not None:
+        params['css_sanitizer'] = CSSSanitizer(allowed_css_properties=styles)
```

Comment:
I feel this easier for my brain to parse.

```suggestion
    if styles:
        params['css_sanitizer'] = CSSSanitizer(allowed_css_properties=all_styles if styles is True else styles)
```